### ISSUE TITLE
[HL1] [REQUEST] Further Data.

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/halflife#1371]
Back in 1999 there was a little addon for Half-Life made officially by Valve named Further Data which included new player models, maps, sprays, a update for HL1 and Uplink. Could it be possible to readd the player models, maps and sprays into Half-Life because some of the maps were eventually added into HL1 by default (GMan, Zombie, Robo, Barney, Scientist, Recon, Gasworks, Crossfire, and Frenzy)

To be added into Half-Life if chosen to be added:
All Sprays

Maps:
doublecross 
rustmill 
team9 
thehill 
xen_dm 

Player Models:
cannibal 
ken 
skeleton 
tmcm 

To download Further Data go here:
http://www.halflifeuplink.com/furtherdata/

**Comment by Pacman1105** [comment]
I think this should be closed since we got most of these in the HL25 update

**Comment by 0Ky** [comment]
Yes, the [25th Anniversary Update](https://www.half-life.com/en/halflife/25th#extras) includes extra contents from the Half-Life: Further Data CD.
- Multiplayer maps (doublecross, rustmill and xen_dm)
- Player models (skeleton and TMCM)
- Many sprays

**Comment by kisak-valve** [comment]
Closing per the last couple comments.
